e2e/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  web:
+    build: ..
+    container_name: web
+    networks:
+      prisoner-content-hub-e2e:
+        aliases:
+          - wayland.prisoner-content-hub.local
+          - berwyn.prisoner-content-hub.local
+          - cookhamwood.prisoner-content-hub.local
+    depends_on:
+      - wiremock
+    restart: always
+    ports:
+      - "3000:3000"
+    env_file: 
+      - "./test-ci.env"
```

Comment:
now two separate test env files - one for running locally and one for inside circle